.gitignore

```diff
@@ -36,6 +36,7 @@ speed-measure-plugin.json
 .history/*
 
 # misc
+/.angular/cache
```

Comment:
should this just be `./angular` like we have in our consumer apps?

Reply:
It was automatically added by the upgrade so I assumed it was fine 🤔

Reply:
Updated just in case 👍